backends/nxp/tests/ir/converter/node_converter/test_constant_pad_nd_converter.py

```diff
@@ -47,37 +39,6 @@ def test_constant_pad_nd_conversion__specific_constant(constant):
     convert_run_compare(edge_program, input_data)
 
 
-@pytest.mark.parametrize("constant", [0.0, 67.28, 42.0, -13.37])
-@pytest.mark.skip(reason="Neutron Converter does not fully convert for NPU")
-def test_constant_pad_nd_quant_conversion__specific_constant(mocker, constant):
```

Comment:
Unrelated change?

Reply:
You are right, this should have been a separate PR to remove irrelevant test. We will pay more attention when rebasing from our development tree.